manim/mobject/svg/brace.py

```diff
@@ -228,3 +228,76 @@ def __init__(
             line_vector = np.array(point_2) - np.array(point_1)
             direction = np.array([line_vector[1], -line_vector[0], 0])
         Brace.__init__(self, Line(point_1, point_2), direction=direction, **kwargs)
+
+
+class ArcBrace(Brace):
+    """Creates a Brace that wraps around an Arc.
+
+    The direction parameter allows the brace to from outside
+    or inside the Arc.
+
+        .. warning::
+            The ArcBraces are smaller for Arcs with smaller radii.
+
+        .. note::
+            The ArcBrace is initially a vertical Brace that takes the dimensions of a line
+            with the length of the Arc but scaled down to match the start and end angles,
+            then it is applied an exponential function after it is shifted
+            according to the radii of the arc.
+            The scale is not applied for arcs with radii smaller than 1 as it will
+            experience overscaling which is unideal.
```

Comment:
```suggestion
    .. warning::
        The ArcBraces are smaller for Arcs with smaller radii.
    
    .. note::
        The ArcBrace is initially a vertical Brace that takes the dimensions of a line
        with the length of the Arc but scaled down to match the start and end angles,
        then it is applied an exponential function after it is shifted
        according to the radii of the arc.
        The scale is not applied for arcs with radii smaller than 1 as it will
        experience overscaling which is unideal.
```